snuba/migrations/migration.py

```diff
@@ -1,10 +1,22 @@
 from abc import ABC, abstractmethod, abstractproperty
 from typing import Sequence
 
+from snuba.migrations.context import Context
 from snuba.migrations.operations import Operation
+from snuba.migrations.status import Status
 
 
-class Migration(ABC):
+class CustomMigration(ABC):
```

Comment:
I would swap the names:
This is the generic class: `Migration`
The subclass in this file is less generic: `MultiStepMigration`

Reply:
Hmm, the idea of flipping the naming was to give the class that should really be used in nearly all cases the generic "Migration" name, and to indicate the other one is a special case that should only be used in rare circumstances.

Reply:
But if you make Migration extend CustomMigration, it means that conceptually a Migration is a CustomMigration and not vice versa. Which would be confusing.

Reply:
Yeah, fair enough. Will update.